client/header/activity-panel/panels/orders.js

```diff
@@ -93,7 +93,7 @@ function OrdersPanel( { orders, isRequesting, isError } ) {
 		const customerUrl = customer.customer_id
 			? getNewPath( {}, '/analytics/customers', {
 					filter: 'single_customer',
-					customer_id: customer.customer_id,
+					customers: customer.customer_id,
```

Comment:
Does the customer link in the orders panel need to be updated for this?

Reply:
@jeffstieler Not quite sure I follow.  The link here needed to be updated to `customers` since the param name shouldn't be `customer_id` (multiple customer IDs can be passed).  Does that answer your question or did I misinterpret?

Reply:
Just re-tested this, it's working as expected - disregard my previous comment. 😛